test/askContainer/treeViews/treeViewProviders/helpViewProvider.test.ts

```diff
@@ -0,0 +1,157 @@
+import * as vscode from "vscode";
+import * as assert from "assert";
+import * as sinon from "sinon";
+
+import { HelpViewProvider } from "../../../../src/askContainer/treeViews/treeViewProviders/helpViewProvider";
+import { HelpView } from "../../../../src/askContainer/treeViews/helpView";
+import { ext } from "../../../../src/extensionGlobals";
+import { PluginTreeItem, CustomResource, ContextValueTypes, Resource } from '../../../../src/runtime/';
+import { HELP_VIEW_ITEMS, EXTERNAL_LINKS } from '../../../../src/constants';
+import { Logger } from '../../../../src/logger';
+
+describe("TreeView_helpViewProvider tests", () => {
+    let helpView: HelpView;
+    let helpViewProvider: HelpViewProvider;
+    let sandbox: sinon.SinonSandbox;
+    let debugSpy: sinon.SinonStub;
+    let verboseSpy: sinon.SinonStub;
+    before(() => {
+        helpView = new HelpView(ext.context);
+        helpViewProvider = new HelpViewProvider(helpView);
+    });
+
+    beforeEach(() => {
+        sandbox = sinon.createSandbox();
+        debugSpy = sandbox.stub(Logger, 'debug');
+        verboseSpy = sandbox.stub(Logger, 'verbose');
+    });
+
+    afterEach(() => {
+        sandbox.restore();
+    });
+
+    describe("getChildren tests", () => {
+        it ('Should add root resources when no element', () => {
+            const expectedGettingStarted = new PluginTreeItem<CustomResource>(
+                HELP_VIEW_ITEMS.GETTING_STARTED, null, vscode.TreeItemCollapsibleState.Collapsed,
+                undefined, undefined, ContextValueTypes.SKILL,
+            );
+            const expectedWhatsNew = new PluginTreeItem<CustomResource>(
+                HELP_VIEW_ITEMS.WHATS_NEW, null, vscode.TreeItemCollapsibleState.None,
+                {
+                    title: 'openUrl',
+                    command: 'ask.container.openUrl',
+                    arguments: [EXTERNAL_LINKS.RELEASE_UPDATES, true, {CommandType: 'RELEASE_UPDATES'}],
+                }, undefined, ContextValueTypes.SKILL,
```

Comment:
This was [recently changed](https://github.com/alexa/ask-toolkit-for-vscode/blob/development/src/askContainer/treeViews/treeViewProviders/helpViewProvider.ts#L51-L57) in the version `2.2.0` and will lead to test failures. Since there is no automated way to run these tests on the repo currently, raised this PR #70 that runs tests during PRs through github actions.

Reply:
rebased my PR on latest development branch and fixed this issue.